packages/pilot/src/containers/Withdraw/index.js

```diff
@@ -116,19 +143,60 @@ class Withdraw extends Component {
     )
   }
 
-  renderCurrentStep () {
+  renderDataStep () {
+    const {
+      maximum,
+      onCancel,
+      t,
+    } = this.props
+
+    return isNil(maximum) || maximum < 100
+      ? buildEmptyState(onCancel, t)
+      : this.renderWithdrawForm
```

Comment:
aqui precisa invocar a função senão não exibe o form no caso de ter saldo =)
```suggestion
      : this.renderWithdrawForm()
```